packages/themes/default/src/components/combobox.scss

```diff
@@ -1,5 +1,8 @@
 @import '../mixins/rem';
-@import '../mixins/alert-wc.scss';
+@import '../mixins/alert-wc';
+
+$option-height: 3rem;
```

Comment:
Bitte überall außer BMF immer rem() verwenden.

Reply:
Wollte ich eigentlich machen, aber die Konsole wirft mir einen Fehler, wenn ich rem in einer Variable verwende

Reply:
Müssten wir trotzdem irgendwie lösen, sonst skaliert das im Zweifelsfall falsch. Sag Bescheid, wenn du hier Hilfe brauchst.